docs/dynamic-plugins/faq.md

```diff
@@ -0,0 +1,154 @@
+# Frequently Asked Questions
+
+## What is Dynamic plugin derived package?
+
+This is a special JavaScript package that is derived from an original plugin package source code.
+You find more information about process of creating derived packages in the [Export Derived Dynamic Plugin Package](export-derived-package.md) document.
+The derived plugins shouldn't be pushed into the [public npm registry](https://www.npmjs.com), but it can be published to a private or internal npm registry.
```

Comment:
```suggestion
The dynamic plugin derived packages shouldn't be pushed into the [public npm registry](https://www.npmjs.com), but it can be published to a private or internal npm registry.
```